packages/components/src/powered-by/index.tsx

```diff
@@ -0,0 +1,82 @@
+import './style.scss';
+
+import colorStudio from '@automattic/color-studio';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import clsx from 'clsx';
+import React from 'react';
+import { JetpackLogo } from '../logos/jetpack-logo';
+import { WooLogo } from '../logos/woo-logo';
+import WordPressWordmark from '../wordpress-wordmark';
+
+const PALETTE = colorStudio.colors;
+const COLOR_WPCOM_BLUE = PALETTE[ 'Blue 50' ];
+const COLOR_WHITE = PALETTE[ 'White' ];
+const COLOR_BLACK = PALETTE[ 'Black' ];
+
+export type PoweredByBrand = 'jetpack' | 'woocommerce' | 'wpcom';
+export type PoweredByVariant = 'color' | 'black' | 'white';
+
+interface PoweredByProps {
+	brand: PoweredByBrand;
+	colorVariant?: PoweredByVariant;
+	className?: string;
+}
+
+const getBrandName = ( brand: PoweredByBrand ) => {
+	switch ( brand ) {
+		case 'jetpack':
+			return 'Jetpack';
+		case 'woocommerce':
+			return 'WooCommerce';
+		case 'wpcom':
+			return 'WordPress.com';
+	}
+};
+
+const getWordPressWordmarkColor = ( colorVariant: PoweredByVariant ) => {
+	switch ( colorVariant ) {
+		case 'white':
+			return COLOR_WHITE;
+		case 'black':
+			return COLOR_BLACK;
+		default:
+			return COLOR_WPCOM_BLUE;
+	}
+};
+
+const PoweredBy = ( { brand, colorVariant = 'color', className }: PoweredByProps ) => {
```

Comment:
Do we ever foresee a need for changing the size of the logo?

Reply:
Basically immediately :), https://github.com/Automattic/wp-calypso/pull/103245